{{ cookiecutter.repo_name }}/src/{{ cookiecutter.__package_name }}/VERSION

```diff
@@ -1 +0,0 @@
-0.0.1-dev
```

Comment:
Im not sure it would not break anything - there was some reason for adding this file :thinking:

Reply:
We can keep it, but according to @mhordynski we should switch to `__version__.py` only.